### DISCUSSION TITLE
My wallpaper is "Overlapping" my real wallpaper and I can't see the icons.

### DISCUSSION THREAD

**Discussion by aguuu27** [rocksdanister/lively#2232] [General]
Hello, sorry for my English. I have this problem when trying to use the program. I wanted to know if you know if there is any kind of solution.

https://github.com/rocksdanister/lively/assets/170221660/8d9ba887-9458-4d71-abc6-3d1fd9341261

I await your response and greetings!

**Comment by rocksdanister** [comment]
Windows is changing things, latest beta version of Lively should work:
https://github.com/rocksdanister/lively-beta/releases

but keep in mind Windows Insider version is not officially supported by Lively and can break in future Windows update.
